### PR TITLE
Print the dependencies in plaintext tree in commander.js

### PR DESCRIPTION
**Pull request by younthu**
- [x] All [tests](https://github.com/jfrog/nuget-deps-tree#building-and-testing-the-sources) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----
`nuget-deps-tree [path to sln file]`, it will print dependencies in json.

or

`nuget-deps-tree --text [path to sln file]`, it will print dependencies in plain text as below:

plainTextFormat

### CODE REVIEW

**Comment by younthu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by younthu** [comment]
Anyone can help to review this please?

File: bin/command.ts [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
+import { NugetDepsTree, Project, DependencyTree } from '../src';
+const { program } = require('commander');
 
-const args = process.argv.slice(2);
-if (args.length !== 1) {
-    throw new Error("Wrong number of arguments. Command accepts only one argument, which is a path to a solution file.")
+program
+  .name('nuget-deps-tree')
+  .description("Nuget Dependency Tree Generator.")
+  .argument("<sln>", "solution file name")
```
Comment (RobiNino) [inline bin/command.ts:8]:
```suggestion
  .argument("<sln>", "solution file path")
```

**Review by RobiNino — CHANGES_REQUESTED** [review]
Thank you very much for your contribution!
I'd appreciate if you could add `"bin/**/*.ts"` to the `format` script in `package.json`, then run `npm run format`.
Thanks!

**Review by RobiNino — APPROVED** [review]

**Comment by RobiNino** [comment]
Thanks again @younthu !
Version 0.4.0 is now released with your new feature.

**Pull request merged**